dev-python/pygtk/pygtk-2.24.0-r5.ebuild

```diff
@@ -0,0 +1,110 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GNOME_TARBALL_SUFFIX="bz2"
+PYTHON_COMPAT=( python2_7 )
+inherit autotools flag-o-matic gnome2 python-r1 virtualx
+
+DESCRIPTION="GTK+2 bindings for Python"
+HOMEPAGE="http://www.pygtk.org/"
```

Comment:
You forgot to update HOMEPAGE, which is the first part of https://bugs.gentoo.org/703052.